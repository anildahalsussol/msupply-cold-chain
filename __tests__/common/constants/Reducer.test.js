import { REDUCER } from '~constants';

describe('Constants: REDUCER', () => {
  it('Matches snapshot', () => {
    expect(REDUCER).toMatchSnapshot();
  });
});
