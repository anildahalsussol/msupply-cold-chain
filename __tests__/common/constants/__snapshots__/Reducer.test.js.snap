// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Constants: REDUCER Matches snapshot 1`] = `
Object {
  "ACKNOWLEDGE_BREACH": "acknowledgeBreach",
  "BATTERY_OBSERVER": "batteryObserver",
  "BLINK": "blink",
  "BLUETOOTH": "bluetooth",
  "BREACH": "breach",
  "BREACH_CONFIGURATION": "breachConfiguration",
  "CHART": "chart",
  "CONSECUTIVE_BREACH": "consecutive",
  "CUMULATIVE_BREACH": "cumulative",
  "DATABASE": "database",
  "DETAIL": "detail",
  "DETAIL_CHART": "detailChart",
  "DETAIL_CUMULATIVE_BREACH": "detailCumulative",
  "DEV": "dev",
  "DEVICE": "device",
  "DOWNLOAD": "download",
  "ENTITIES": "entities",
  "LOG_TABLE": "logTable",
  "NEW_SENSOR": "new_sensor",
  "PERMISSION": "permission",
  "PROGRAM": "program",
  "REPORT": "report",
  "SCAN": "scan",
  "SENSOR": "sensor",
  "SENSORS": "sensors",
  "SENSOR_DETAIL": "sensorDetail",
  "SENSOR_STATUS": "sensorStatus",
  "SETTING": "setting",
  "SYNC": "sync",
  "TEMPERATURE_LOGS": "temperatureLogs",
  "UPDATE": "update",
}
`;
